frontend/src/Components/Form/MultiSelectInputSelectedValue.js

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Label from 'Components/Label';
+import EnhancedSelectInputSelectedValue from './EnhancedSelectInputSelectedValue';
+import styles from './MultiSelectInputSelectedValue.css';
+
+function MultiSelectInputSelectedValue(props) {
+  const {
+    value,
+    values,
+    ...otherProps
+  } = props;
+
+  return (
+    <EnhancedSelectInputSelectedValue
+      className={styles.selectedValue}
+      {...otherProps}
+    >
+      <div className={styles.valueText}>
+        {
+          value.map((key, index) => {
+            const v = values.find((i) => i.key === key);
```

Comment:
Since this will run every time it's rendered if values is going to have a lot of values having an object with the key being the key and the value being the entire object would be better for performance, not sure it'll get to that point, but something to consider.